tests/sentry/search/events/test_filter.py

```diff
@@ -840,12 +840,10 @@ def test_negated_wildcard(self):
 
     def test_escaped_wildcard(self):
         assert get_filter("release:3.1.\\* user.email:\\*@example.com").conditions == [
-            [["match", ["release", "'(?i)^3\\.1\\.\\*$'"]], "=", 1],
-            [["match", ["user.email", "'(?i)^\\*@example\\.com$'"]], "=", 1],
-        ]
-        assert get_filter("release:\\\\\\*").conditions == [
-            [["match", ["release", "'(?i)^\\\\\\*$'"]], "=", 1]
+            ["release", "=", "3.1.*"],
+            ["user.email", "=", "*@example.com"],
         ]
+        assert get_filter("release:\\\\\\*").conditions == [["release", "=", "\\\\*"]]
```

Comment:
These conditions are equal to what they were before. Previously, it was using a regex to match the literal, and now it's just using a plain equals.